src/main/java/org/example/bookweb/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,65 @@
+package org.example.bookweb.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.example.bookweb.dto.shopping.cart.AddCartItemDto;
+import org.example.bookweb.dto.shopping.cart.ShoppingCartResponseDto;
+import org.example.bookweb.dto.shopping.cart.UpdateCartItemDto;
+import org.example.bookweb.service.shopping.cart.ShoppingCartService;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Shopping cart management",
+        description = "Endpoints for managing shopping cart with cart items")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/api/cart")
```

Comment:
```suggestion
@RequestMapping("/cart")
```
Add `server.servlet.context-path=/api` in application.properties